### PR TITLE
[Admin] Create product layout

### PR DESCRIPTION
**Pull request by at-khadinh**
## Description
Add Layout for Create product page
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/admin/pages/products/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@extends('admin.layout.master')
+@section('title', __('product.create.title'))
+@section('content')
+<div class="right_col" role="main" style="min-height: 1162px;">
```
Comment (at-xinhnguyen) [inline resources/views/admin/pages/products/create.blade.php]: move `đoạn css inline` này vào `css file` đi em

File: resources/lang/en/product.php [hunk 2]
```diff
@@ -16,5 +16,18 @@
...
+        'title' => 'Create Product',
+        'table-title' => 'Create Product',
+        'category' => 'Category',
+        'name' => 'Name',
+        'description' => 'Description',
+        'price' => 'Price',
+        'quantity' => 'Quantity',
+        'image' => 'Image',
+        'file-input' => 'File Input',
+        'cancel' => 'Cancel',
```
Comment (at-xinhnguyen) [inline resources/lang/en/product.php]: `cancel` em nên bỏ vào en/common.

File: resources/views/admin/pages/products/create.blade.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      <div class="x_panel">
+        <div class="x_title">
+          <h2>@lang('product.create.table-title')</h2>
+          <div class="clearfix"></div>
+        </div>
+        <div class="x_content">
+          <br />
+          <form id="demo-form2" data-parsley-validate class="form-horizontal form-label-left">
+
+            {{ csrf_field() }}
```
Comment (at-xinhnguyen) [inline resources/views/admin/pages/products/create.blade.php]: 5.6 em có thể dùng @csrf  cho gọn ^_^. để như em vẫn okey

**Review by ghost — CHANGES_REQUESTED** [review]
please check my cmt

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**